apiary.apib

```diff
@@ -3228,7 +3233,9 @@ Get a list of meetings.
                     + Members
                         + open
                         + done
-
+                + recurrence (object, nullable)
+                    + type: `recurrence` (string)
```

Comment:
Q: We don't expose this resource anywhere do we?

To which type refers this id then? Without exposing the resource I think we shouldn't include this property yet.

Reply:
> Q: We don't expose this resource anywhere yet, do we?
> 
> To which type refers this id then? Without exposing the resource I think we shouldn't include this property yet.

🤔 Right. the goal is to be able to list recurring meetings from a recurrence id, so providing the recurrence (definition) isn't necessary for now.
We can omit the type property for now and provide just the id, which is the necessary info to have.

Reply:
Okay, where would that id then come from? From existing meetings which return the recurrence?

Though, I wouldn't really remove the type if we'd still expose the id.